lib/fastlane/plugin/wpmreleasetoolkit/actions/bundletool/bundletool_generate_universal_signed_apk_action.rb

```diff
@@ -0,0 +1,104 @@
+module Fastlane
+  module Actions
+    class BundletoolGenerateUniversalSignedApkAction < Action
+      def self.run(params)
+        aab_path = params[:aab_path]
```

Comment:
Given the way we typically generate the `.aab` to be used here is by calling the `gradle` action in our Fastfile first, it would be nice to default [to `SharedValues::GRADLE_AAB_OUTPUT_PATH`](https://docs.fastlane.tools/actions/gradle/#lane-variables) here if the user didn't provide an `aab_path` explicitly during invocation, so that it uses the value provided by the previous invocation of the `gradle` action if any,

You can take some inspiration from [the official `upload_to_play_store` fastlane action here](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/upload_to_play_store.rb#L19-L29) which uses a similar trick so that when `upload_to_play_store` is called just after calling `gradle`, you don't have to pass a value for those parameters explicitly and instead it will use the output values of the previous `gradle` command.

Reply:
Thanks, that's a great suggestion! I've implemented it in 16e5050557dcad2312d1b7db19c7b1ddfa71119b